lib/grandpa/grandpa.go

```diff
@@ -846,6 +852,23 @@ func (s *Service) createJustification(bfc common.Hash, stage Subround) ([]Signed
 		return nil, err
 	}
 
+	for _, votes := range eqv {
+		for _, vote := range votes {
+			var signedVote SignedVote = *vote
```

Comment:
nit why not just

```suggestion
			signedVote := *vote
```

You could also move the derefencing inline with the `append` since `signedVote.Vote.Hash` doesn't need to be dereferenced.